clock/test/controllers/reports_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class ReportsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @report = reports(:one)
+  end
+
+  test "should get index" do
+    get reports_url
+    assert_response :success
+  end
+
+  test "should get new" do
+    get new_report_url
+    assert_response :success
+  end
```

Comment:
Worth adding another spec for this method for ensuring that it redirects to the `edit_path` when there is a pending-report for an employee